### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by pankajkoti**
Add CODEOWNERS so that they are automatically requested for review when someone opens a pull request.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+@pankajastro @pankajkoti @tatiana
```
Comment (pankajkoti) [inline CODEOWNERS:1]:
```suggestion
@astronomer/astro-dagfactory-codeowners
```

Check instead if such an alias/group with adding those team members can be created & used here

Comment (tatiana) [reply CODEOWNERS:1]: I just asked @phanikumv for us to have a single GH team for the OSS Integrations team, and then we could assign this both here and in the other repos. To maintain a team per project would be just additional maintenance overhead

Comment (pankajkoti) [reply CODEOWNERS:1]: thanks @tatiana . @phanikumv could you please let me know once you create that team and the name of the team so I can add it here and in other repos, too?

Comment (phanikumv) [reply CODEOWNERS:1]: Added `oss-integrations` github team 

https://github.com/orgs/astronomer/teams/oss-integrations/members

Comment (pankajkoti) [inline CODEOWNERS:1]:
```suggestion
@oss-integrations
```

Comment (pankajkoti) [reply CODEOWNERS]: Thank you @phanikumv. Applied the change.

**Review by tatiana — APPROVED** [review]
Looks great, thanks a lot, @pankajkoti !

**Pull request merged**